flocker/cli/script.py

```diff
@@ -65,6 +65,12 @@ class DeployOptions(Options):
     optParameters = [
         ["port", "p", REST_API_PORT,
          "The REST API port on the server.", int],
+        ["cafile", None, b"cluster.crt",
```

Comment:
Full stops on descriptions (I don't mind either way as long as it is consistent, and the existing ones have full stops).